client/lib/site/index.js

```diff
@@ -212,10 +212,6 @@ Site.prototype.fetchUsers = function() {
 	}.bind( this ) );
 };
 
-Site.prototype.isUpgradeable = function() {
-	return this.capabilities && this.capabilities.manage_options;
-};
-
```

Comment:
You may want to remove the mention to this method from the [`README.md` file](https://github.com/Automattic/wp-calypso/blob/d9edb63952c6335f9fa8ceab64061af27b6f9694/client/lib/site/README.md).